sqlserver/tests/test_statements.py

```diff
@@ -337,6 +353,43 @@ def _obfuscate_sql(sql_query, options=None):
     assert metric['dd_tables'] == expected_metadata_payload['tables']
     assert metric['dd_commands'] == expected_metadata_payload['commands']
 
+    # Setup activity for metadata testing
+    blocking_query = "INSERT INTO ϑings WITH (TABLOCK, HOLDLOCK) (name) VALUES ('puppy')"
+    fred_conn = _get_conn_for_user('fred')
```

Comment:
I refactored the way I did this in the activity tests [right here](https://github.com/DataDog/integrations-core/blob/master/sqlserver/tests/test_activity.py#L58-L92) to be simpler and more deterministic. Wanna update your code to follow the same pattern?